### PR TITLE
Flake8 test fixes

### PR DESCRIPTION
**Pull request by balloob**
Our test file was not passing pep8.

Should we include testing it to travis?

### CODE REVIEW

**Comment by bboe** [comment]
> Should we include testing it to travis?

We certainly could.

**Comment by balloob** [comment]
Done.

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,8 @@
 [wheel]
-universal = 1
\ No newline at end of file
+universal = 1
+
+[flake8]
```
Comment (bboe) [inline setup.cfg]: I didn't know this configuration could go in here. Cool!

File: setup.cfg [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+exclude = examples, docs
+
+[pep257]
```
Comment (bboe) [inline setup.cfg]: We should probably switch to pydocstyle (the renamed version), but we can do that in a different PR.
